tests/Agency/StorePerformanceTest.cpp

```diff
@@ -0,0 +1,423 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
```

Comment:
Should be 2021 by now.
```suggestion
/// Copyright 2014-2021 ArangoDB GmbH, Cologne, Germany
```